aptos-move/move-examples/post_mint_reveal_nft/Move.toml

```diff
@@ -1,21 +1,13 @@
 [package]
-name = "test_package"
+name = "PostMint"
 version = "0.0.0"
-upgrade_policy = "compatible"
 
 [dependencies]
 AptosFramework = { local = "../../framework/aptos-framework" }
 AptosToken = { local = "../../framework/aptos-token" }
-DataStructure = { local = "../data_structures" }
 
 [addresses]
 aptos_framework = "0x1"
-# the source_addr and post_mint_reveal_nft here are for unit test purposes.
-# in production, we should change them:
-# - the source_addr should be the address of the account that creates the resource account
-# - the post_mint_reveal_nft should be put as "_" if we publish the module by using the CLI command
-# "aptos move create-resource-account-and-publish-package ..." because the CLI will automatically link
-# the address to the package. If we are not using the CLI, the post_mint_reveal_nft should be the address
-# of the resource account which is also the module publisher.
-source_addr = "0xcafe"
-post_mint_reveal_nft = "0xc3bb8488ab1a5815a9d543d7e41b0e0df46a7396f89b22821f07a4362f75ddc5"
\ No newline at end of file
+# the post_mint_reveal_nft here are for unit test purposes.
+# in production, we should change it to "post_mint_reveal_nft = "_""
+post_mint_reveal_nft = "0xcafe"
```

Comment:
why not add [dev_addresses] and put this there for unit test? then you can change this to _

Reply:
I don't think `aptos move test` with flag `-d` is working yet? in that case running the unit tests with `aptos move test` would fail - lmk if I am wrong tho

Reply:
You don't need the -d flag. It'll just pick up the address from dev_addresses. I learned this trick from @jjleng

Reply:
I think [bowenyang007](https://github.com/bowenyang007) is right. We did it for the minting contract https://github.com/aptos-labs/token/blob/main/contracts/mint_nft/Move.toml#L20-L22 also. I don't think you need to provide the `-d` flag.

Reply:
kk makes sense - just updated

Reply:
realized that previously it gave me the error bc I deleted the `post_mint = "_"` line under `[addresses]`